src/components/Link/docs/Link.md

```diff
@@ -0,0 +1,21 @@
+Links!
```

Comment:
Might want a tad more about why one may need/want a Link component :D

Reply:
LOL yes, forgot to fill in this section.